spec/index.bs

```diff
@@ -491,36 +496,27 @@ enum GPUCompareFunction {
 Resource Binding {#binding}
 ===========================
 
-## GPUPipelineLayout ## {#pipeline-layout}
-
-<script type=idl>
-interface GPUPipelineLayout : GPUObjectBase {
-};
-</script>
-
-### Creation ### {#pipeline-layout-creation}
-
-<script type=idl>
-dictionary GPUPipelineLayoutDescriptor : GPUObjectDescriptorBase {
-    required sequence<GPUBindGroupLayout> bindGroupLayouts;
-};
-</script>
-
 ## GPUBindGroupLayout ## {#bind-group-layout}
 
+A {{GPUBindGroupLayout}} defines a set of shader resource bindings and their relationships to shader stages.
```

Comment:
This should probably be rewritten. Vulkan spec has it as:
> The layout is used both for determining the resources that need to be associated with the descriptor set, and determining the interface between shader stages and shader resources.

Perhaps, we could phrase it as: "defines an interface between the resources bound to a pass via {{GPUBindGroup}} and the shaders"

Reply:
Re-worded this a bit. PTAL